Audit flaky tests

### Builds are flailing randomly due to flaky tests

Looking back at flailing builds for the last couple of weeks.

- `Pull request #1789 - Fixing functionality to delete a pull request comment`
`GitHub.InlineReviews.UnitTests.Services.PullRequestSessionManagerTests+TheGetLiveFileMethod.AddsNewReviewCommentToThread` ❌  

- `Pull request #1791 - Fix icon background in PullRequestReviewAuthoringView.`
`RepositoryCreationViewModelTests+TheLicensesProperty.IsPopulatedByTheModelServiceAsync` ❌

- `Pull request #1789 - Adding functionality to delete a pull request comment`
`error CS7036: There is no argument given that corresponds to the required formal parameter 'pullRequestId' of 'IPullRequestSession.DeleteComment(int, int)' ` ❌

- `Make <summary> xmldoc elements a single sentence`
`GitHub.InlineReviews.UnitTests.Services.PullRequestSessionManagerTests+TheGetLiveFileMethod.AddsNewReviewCommentToThread` ❌

- `Merge branch 'master' into refactor/pr-list-0.6-update`
`GitHub.InlineReviews.UnitTests.Services.PullRequestSessionManagerTests+TheGetLiveFileMethod.AddsNewReviewCommentToThread` ❌
TrackingTests.Removing ❌

It appears that the  majority of broken builds are due to the flaky test `TheGetLiveFileMethod.AddsNewReviewCommentToThread`.

### Related

- https://github.com/github/VisualStudio/issues/1722 Flaky and failing tests

--- a/test/GitHub.InlineReviews.UnitTests/Services/PullRequestSessionManagerTests.cs
+++ b/test/GitHub.InlineReviews.UnitTests/Services/PullRequestSessionManagerTests.cs
@@ -555,7 +555,7 @@ Line 4";
                 }
             }
 
-            [Test]
+            [Test, Ignore("Flaky test, see https://github.com/github/VisualStudio/issues/1795")]
             public async Task AddsNewReviewCommentToThread()
             {
                 var baseContents = @"Line 1

